specification/healthcareapis/resource-manager/readme.md

```diff
@@ -136,6 +136,31 @@ regenerate-manager: true
 generate-interface: true
 ```
 
+## CLI Tools
```

Comment:
can this been added into a separate file like readme.cli.md, to avoid confusion to PR author and reviewers?